test/target_ubuntu_test.go

```diff
@@ -24,11 +25,17 @@ func withPackageOverride(oldPkg, newPkg string) func(cfg *testLinuxConfig) {
 }
 
 func debLinuxTestConfigFor(targetKey string, cfg *distro.Config, opts ...func(*testLinuxConfig)) testLinuxConfig {
+	var sysextTarget string
+	if slices.Contains(cfg.BuilderPackages, "erofs-utils") {
```

Comment:
Maybe noble can pass an opt here that sets sysextTarget instead and only run the sysext test when the target is set?

Reply:
I've done this in a slightly different way now, but I think it's close enough to what you wanted.